spec/localeapp/cli/install_spec.rb

```diff
@@ -192,6 +249,23 @@
   end
 end
 
+describe Localeapp::CLI::Install::DefaultInstaller, '#write_apikey_to_dotenv' do
+  let(:output) { StringIO.new }
+  let(:key) { 'APIKEY' }
+  let(:installer) { Localeapp::CLI::Install::DefaultInstaller.new(output) }
+
+  it "creates a configuration file containing just the api key" do
+    installer.key = key
+    file = double('file')
+    expect(file).to receive(:write).with <<-CONTENT
+
+LOCALEAPP_API_KEY=APIKEY
+CONTENT
+    expect(File).to receive(:open).with('.env', 'a').and_yield(file)
+    installer.write_apikey_to_dotenv
+  end
+end
+
```

Comment:
I'm making a single comments for all those new specs you added on `Localeapp::CLI::Install::DefaultInstaller`.

I think it shows us that this class already has too much responsibilities, and/or issues, because we cannot write "good" spec (those new ones rely heavily on mocks, and are too much coupled with other classes).

Maybe a solution would be to extract some logic from this class (the new one you add in this PR, and maybe other if required). IMO we should try to obtain a version where we don't need to mock the installer at the same time that we test it. I hope I'm being clear.